routes/degree_days.py

```diff
@@ -252,52 +301,80 @@ async def fetch_dd_point_data(x, y, cov_id, horp, start_year, end_year):
     return point_data_list
 
 
-def create_csv(data_pkg, var_ep, place_id=None, lat=None, lon=None):
+async def fetch_di_point_data(x, y, cov_id):
+    """Run the async design index data request for a single point
+    and return data as json
+
+    Args:
+        lat (float): latitude
+        lon (float): longitude
+        cov_id (str): design_thawing_index or freezing_thawing_index
+
+    Returns:
+        JSON-like dict of data at provided latitude and longitude
+    """
+    request_str = generate_wcs_getcov_str(x, y, cov_id)
+    point_data_list = await fetch_data([generate_wcs_query_url(request_str)])
+    return point_data_list
+
+
+def create_csv(data_pkg, cov_id_str, place_id=None, lat=None, lon=None):
     """Create CSV file with metadata string and location based filename.
     Args:
         data_pkg (dict): JSON-like object of data
-        var_ep (str): heating, below_zero, thawing_index, or freezing_index
+        cov_id_str (str): coverage id string
         place_id: place identifier (e.g., AK124)
         lat: latitude for points or None for polygons
         lon: longitude for points or None for polygons
     Returns:
         CSV response object
     """
-
-    fieldnames = [
-        "model",
-        "year",
-        "variable",
-        "value",
-    ]
+    if cov_id_str in ["design_thawing_index", "design_freezing_index"]:
+        fieldnames = [
+            "model",
+            "year",
+            "variable",
+            "value",
+        ]
+    else:
+        fieldnames = [
+            "model",
+            "year",
+            "variable",
+            "value",
+        ]
```

Comment:
Do we need this change? It looks like `fieldnames` is the same in both cases of the `if` statement?

Reply:
Good catch! Somewhere along the way while I was working on this the fieldnames became the same for degree days vs. design index. I'll get this fixed.

Reply:
Fixed in 160f8ec63c33670c5f7c329fe9a4086e55bed377.